packages/tools/client-debugger/client-debugger-view/src/test/app/index.tsx

```diff
@@ -40,7 +40,7 @@ function DevToolsView(): React.ReactElement {
 				zIndex: "2",
 				backgroundColor: "lightgray", // TODO: remove
 			}}
-			defaultSize={{ width: 500, height: "100%" }}
+			defaultSize={{ width: 550, height: "100%" }}
```

Comment:
This is just to avoid wrapping in the container summary view with the default size of the inline UI.

Goes from this:

![image](https://user-images.githubusercontent.com/716334/232808841-4fb9294e-415f-4ab7-8fa4-2c2b63685c79.png)

To this:

![image](https://user-images.githubusercontent.com/716334/232808966-e63d5e96-b6e8-410f-acfe-b7c32c270b57.png)

Reply:
This won't help us in the devtools view, though. We should probably be prioritizing styling for that default view size if we can IMO.